subworkflows/local/read_coverage.nf

```diff
@@ -194,18 +144,19 @@ workflow READ_COVERAGE {
     ch_versions             = ch_versions.mix(BEDTOOLS_GENOMECOV.out.versions)
 
     //
-    // MODULE: SORT THE PRIMARY BED FILE
+    // LOGIC: BED2BIGWIG TAKES SORTED COVERAGE BED FILE
```

Comment:
Would this be a MODULE rather than LOGIC block?

Reply:
I think LOGIC, it is a module but to emphasizes why we have to sort it multiple times?